.env.template

```diff
@@ -174,9 +175,10 @@ GDS_BFF_MAINNET_MEMBERS_TIMEOUT=5s
 GDS_BFF_MAINNET_MEMBERS_MTLS_CERT_PATH=
 GDS_BFF_MAINNET_MEMBERS_MTLS_POOL_PATH=
 
-GDS_BFF_MAINNET_ADMIN_ENDPOINT=http://localhost:5434
-GDS_BFF_MAINNET_ADMIN_AUDIENCE=http://localhost:4433
-GDS_BFF_MAINNET_ADMIN_TOKEN_KEYS=
+GDS_BFF_MAINNET_DATABASE_URL=trtl://localhost:4436
+GDS_BFF_MAINNET_DATABASE_INSECURE=true
+GDS_BFF_MAINNET_DATABASE_CERT_PATH=
+GDS_BFF_MAINNET_DATABASE_POOL_PATH=
```

Comment:
Note: Updated environment variables

Reply:
Were these updated in docker-compose.yaml?